toml/value.hpp

```diff
@@ -1927,36 +1927,33 @@ operator>=(const basic_value<C, T, A>& lhs, const basic_value<C, T, A>& rhs)
 template<typename C, template<typename ...> class T, template<typename ...> class A>
 inline std::string format_error(const std::string& err_msg,
         const basic_value<C, T, A>& v, const std::string& comment,
-        std::vector<std::string> hints = {},
-        const bool colorize = TOML11_ERROR_MESSAGE_COLORIZED)
```

Comment:
While I actually don't know who uses the `colorize` variable instead of the macro, since this variable can be set manually, it is better to keep this variable for backward compatibility (we can remove this value in a later release).

We can use the `enable` and `disable` functions depending on the value of the `colorize` variable to force turn the coloring on and off.

Reply:
Ah, totally forgot this also brings breaking changes! Thank you for picking it up!

I will fix the code as reviewed :)